local-dev/templates/sqlproxy.env

```diff
@@ -0,0 +1,4 @@
+GOOGLE_PROJECT=broad-dsde-dev
```

Comment:
The name of the google project was part of the secret before, but I think it's ok for this to not be secret